intrinsic_funcs.md

```diff
@@ -197,6 +197,65 @@ vfloat64m1_t vle64_v_f64m1 (const float64_t *base, size_t vl);
 vfloat64m2_t vle64_v_f64m2 (const float64_t *base, size_t vl);
 vfloat64m4_t vle64_v_f64m4 (const float64_t *base, size_t vl);
 vfloat64m8_t vle64_v_f64m8 (const float64_t *base, size_t vl);
+vint8mf8_t vle8_v_i8mf8_t (vint8mf8_t dest, const int8_t *base, size_t vl, uint8_t ta);
```

Comment:
The implementation in clang for the _mt intrinsics is using size_t not uint8_t for policy if I'm reading RISCVVEmitter.cpp correctly.

Reply:
You are right. I need to update it to size_t.